cli/azd/pkg/contracts/envelope.go

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package contracts
```

Comment:
Will we suffer from the package name squandering due to multiple packages all named `contracts`? I think we need to start shifting towards creating "application layer" packages.

A single `azdcontracts` package in `pkg` makes the most sense to me. The reason to prefix with `azd` is that `contracts` could collide with other packages, for example: `errors` will collide with `pkg/errors`

Reply:
Yeah - this is a good idea.  I will move this code around in a follow up PR.  I'd like to land this now to unblock the release.